modules/dnn/test/test_layers.cpp

```diff
@@ -743,6 +743,32 @@ TEST_F(Layer_RNN_Test, get_set_test)
     EXPECT_EQ(shape(outputs[1]), shape(nT, nS, nH));
 }
 
+TEST(Layer_MHARoPe_Test_Accuracy_with_, Pytorch)
```

Comment:
It'll be great, if the test imports ONNX, but not create the attention layer in code. It allows to check parser and ensure that the implementation handles feasible graph.

Reply:
This is a little tricky due to compatibility issues with onnx runtime on my dev machine (apple silicon)
I have just checked, do_rotary flag is ignored. I 
so i can't produce a reference output of the corresponding onnx (with `do_rotary=1`) model locally.